src/js/models/layer.js

```diff
@@ -37,6 +37,29 @@ define(function (require, exports, module) {
         Text = require("./text");
 
     /**
+     * Possible smart object types for a layer
```

Comment:
I thought about that... but these are values that we defined, unlike layer kinds.